synapse/storage/databases/main/events_worker.py

```diff
@@ -259,7 +283,7 @@ def get_events_as_list(
                 omits rejected events from the response.
 
         Returns:
-            Deferred[list[EventBase]]: List of events fetched from the database. The
+            list[EventBase]: List of events fetched from the database. The
```

Comment:
Can we not move this to a type hint since the function is already typed?

Reply:
Not sure why I didn't do that! 👍